Marlin/Makefile

```diff
@@ -37,12 +37,14 @@
 # from the commandline with "make HARDWARE_MOTHERBOARD=71" for example
 
 # This defined the board you are compiling for (see boards.h for the options)
-HARDWARE_MOTHERBOARD ?= 11
+HARDWARE_MOTHERBOARD ?= 33
 
 # Arduino source install directory, and version number
 # On most linuxes this will be /usr/share/arduino
+# On Mac OsX its probaly /Applications/Arduino.app/Contents/Resources/Java
```

Comment:
Spelling correction:

    # On OS X it's probably /Applications/Arduino.app/Contents/Resources/Java